cloudify_awssdk/autoscaling/resources/autoscaling_group.py

```diff
@@ -168,7 +168,10 @@ def create(ctx, iface, resource_config, **_):
             ctx.instance,
             SUBNET_TYPE_DEPRECATED,
             subnet_list)
+
     if subnet_list:
+        # Remove any duplicate items from subnet list
+        subnet_list = list(set(subnet_list))
```

Comment:
Maybe we should chck that it needs to be converted to a list?

Reply:
I think we should converted to list, since the data type for `VPCZoneIdentifier` should be python `list`